mailpoet/lib/Automation/Engine/Storage/AutomationStorage.php

```diff
@@ -65,6 +68,34 @@ public function updateAutomation(Automation $automation): void {
     $this->insertAutomationTriggers($automation->getId(), $automation);
   }
 
+  /**
+   * @param int $automationId
+   * @return VersionDate[]
+   * @throws \Exception
+   */
+  public function getAutomationVersionDates(int $automationId): array {
+    $versionsTable = esc_sql($this->versionsTable);
```

Comment:
Maybe you need to wrap this WP function and not use it directly? Same applies to a few other `esc_sql` calls in this PR.